stdlib/2/urllib.pyi

```diff
@@ -1,7 +1,7 @@
 from typing import Any, AnyStr, IO, Mapping, Sequence, Text, Tuple, Union
 
-def url2pathname(pathname: str) -> str: ...
-def pathname2url(pathname: str) -> str: ...
+def url2pathname(pathname: AnyStr) -> AnyStr: ...
+def pathname2url(pathname: Text) -> Text: ...
```

Comment:
`pathname2url()` seems to just call `quote()`, which is typed as 
```python
def quote(s: AnyStr, safe: Text = ...) -> AnyStr: ...
```
Looking at `quote()`, it can return `unicode` or `str` if `unicode` is supplied, but only `str` if a `str` is supplied. Therefore I believe, using `AnyStr` here is a bit better.

Reply:
@mr-c

Reply:
Fixed!